reference/tax_settings.v3.yml

```
@@ -178,6 +190,18 @@ components:
           default: true
           type: boolean
           description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
+          default: false
+          type: boolean
+          description: When enabled, geolocation data will be used to match the guest shopper with a relevant country-level tax zone for tax estimation on the storefront. The geolocation data will be used to determine the corresponding tax zone provided in basic tax - once the corresponding tax zone is found, the tax estimation will be applied to the store. When this setting is disabled, the tax estimation will be based on the guest shopper tax zone, the guest shopper tax zone can be reconfigured by updating guest_shopper_tax_zone_id or store_tax_zone_id.
```

The second sentence looks like a repeat of the first - thoughts?

> the tax estimation will be applied to the store

I'm not sure I'd describe tax estimation as applying to a store.

I think we can simplify this to:

```suggestion
          description: When enabled, geolocation data will be used to match the guest shopper with a relevant country-level tax zone for tax estimation on the storefront. When disabled, storefront tax estimation will always use the guest shopper tax zone.
```